apps/escuelafrontend/prisma/schema.prisma

```diff
@@ -0,0 +1,63 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  // referential integrity is to disable foreign keys and relations in prisma
+  // see: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-integrity
+  previewFeatures = ["referentialIntegrity", "interactiveTransactions"]
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+  // shadow is only used for running prims migrations
+  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
```

Comment:
we can drop this in favor of using `prisma db push` and letting planetscale work as intended with branches